clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -1546,7 +1546,7 @@ Expected<StringRef> clang(ArrayRef<StringRef> InputFiles, const ArgList &Args,
     CmdArgs.push_back("-Wl,--no-undefined");
 
   if (IsSYCLKind && Triple.isNVPTX())
-    CmdArgs.push_back("-Wl,--lto-emit-asm");
+    CmdArgs.push_back("-S");
```

Comment:
when was lto removed to cuda kernels? was that done upstream or by us downstream? thx

Reply:
Then it was reintroduced to amd gpus only through this PR: https://github.com/llvm/llvm-project/pull/129118

Reply:
ok yeah i think nvidia doesnt use lto, thanks